istio-telemetry/mixer-telemetry/values.yaml

```diff
@@ -53,6 +53,15 @@ mixer:
     stackdriver:
       enabled: false
 
+      auth:
+        appCredentials: false
+        apiKey: ""
+        serviceAccountPath: ""
+
+      tracer:
```

Comment:
Tracing is defined in like 5 places already... I have no clue how to configure it. Why is this a new field instead of using the existing values?

Reply:
I think this is the right place, since mixer is sending the requests.

The others are in global since they have to be injected ( sidecar is sending the data )

Reply:
We have:
* enableTracing
* tracer
* tracing.enabled
* proxy.tracer

and now mixer.tracer

Maybe this one is right and the others are wrong, but our tracing config is crazy... worse than logging I think

Reply:
When I decided to enable tracing, I also got confused about all those config options. For a while, I thought standing up zipkin was required for enabling tracing since all proxies are set up to send traces to zipkin.istio-system by default and assumed that zipkin will be responsible for sending traces to Stackdriver -which definitely was not true. 

Maybe adding sidecar to stackdriver support might make this less confusing opposed to mixer sending the data to SD.